operators/src/processing/interpolation/mod.rs

```diff
@@ -44,14 +51,9 @@ pub enum InterpolationMethod {
 #[snafu(visibility(pub(crate)), context(suffix(false)), module(error))]
 pub enum InterpolationError {
     #[snafu(display(
-        "The query resolution ({:?}) must be smaller than the input resolution ({:?})",
-        query_resolution,
-        input_resolution
+        "The input resolution was defined as `native` but the source resolution is unknown.",
```

Comment:
Please check the errors

```suggestion
        "The input resolution was defined as `source` but the source resolution is unknown.",
```